apollo-normalized-cache/src/commonMain/kotlin/com/apollographql/apollo/cache/normalized/MemoryCache.kt

```diff
@@ -0,0 +1,99 @@
+package com.apollographql.apollo.cache.normalized
+
+import com.apollographql.apollo.cache.ApolloCacheHeaders
+import com.apollographql.apollo.cache.CacheHeaders
+import com.apollographql.apollo.cache.normalized.internal.LruCache
+import com.apollographql.apollo.cache.normalized.internal.Platform
+import okio.internal.commonAsUtf8ToByteArray
+import kotlin.reflect.KClass
+
+class MemoryCache(
```

Comment:
Should we remove `MapNormalizedCache` and only keep one `MemoryNormalizedCache` ?

Reply:
Yeah I guess I forgot to do this.